.drone.yml

```diff
@@ -0,0 +1,120 @@
+---
+kind: pipeline
+name: default
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: docker-build
+    pull: if-not-exists
+    image: docker:17.09.1
+    commands:
+      - docker build -t hocs-toolbox .
+    environment:
+      DOCKER_HOST: tcp://172.17.0.1:2375
+    when:
+      branch:
+        - master
+        - refs/tags/*
+      event:
+        - push
+        - tag
+
```

Comment:
We're building and pushing as the next step, so we don't need this `docker-build` step too.

Reply:
Removed.